docs/concepts/operator.md

```diff
@@ -232,6 +232,106 @@ Please refer to [this document](../usage/shoot_credentials_rotation.md#gardener-
 
 ⚠️ Rotation of static `ServiceAccount` secrets is not supported since the `kube-controller-manager` does not enable the `serviceaccount-token` controller.
 
+## Upgrading An Existing Gardener Landscape To `gardener-operator`
```

Comment:
I see that we are not completely consistent with regards to casing in section titles. My preference would be to use lower case for filler words like "a/an" or "to". However, this is not done consistently in this document. Therefore, I leave it up to you.